scripts/ci/prep-local-docker-registry-credentials

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env bash
 
 set -euvo pipefail
+
+# Creates Basic Authentication Context
+pushd tests/functional/artifacts/registry/
+
 echo "Generating Local Docker Registry Artifacts at $(pwd)"
 mkdir -p auth certs
 
-# Creates Basic Authentication Context
 docker run --rm --entrypoint htpasswd registry:2.7.0 -Bbn "$ANCHORE_TEST_DOCKER_REGISTRY_USER" "$ANCHORE_TEST_DOCKER_REGISTRY_PASS" > ./auth/htpasswd
-openssl req -newkey rsa:4096 -nodes -sha256 -keyout ./certs/domain.key -x509 -days 365 -out ./certs/domain.crt -subj "/C=US/ST=California/L=Santa Barbara/O=Anchore/CN=functional_test"
-docker build -f tests/functional/artifacts/registry/Dockerfile -t anchore-local-docker-registry:latest .
-rm -rf auth certs
```

Comment:
don't want to clean up the directories anymore?

Reply:
correct --this is one of the cleanups. Since we've moved to a VM machine type in CircleCI we can leverage (and as of this PR are leveraging) docker volumes instead of building an image with ephemeral secrets.

These directories are now persisted at `tests/functional/artifacts/registry/` and are mounted into the registry container as a volume, so they are never cleaned up or removed (which we could optionally do after `compose-down`).